active_record_cars/car.rb

```diff
@@ -0,0 +1,15 @@
+require 'active_record'
+
+class Car < ActiveRecord::Base
+	establish_connection adapter: 'sqlite3',
+						 database: 'cars.db'
+
+	def luxury
+		if Car.price > 30
```

Comment:
that should be self.price > 30, or just price > 30, but not Car.price bc .price isn't a class method.
Also, if price > 30, then what?